.jenkinsci/linux-post-step.groovy

```diff
@@ -1,6 +1,23 @@
 def linuxPostStep() {
   timeout(time: 600, unit: "SECONDS") {
     try {
+      // stop write core dumps
+      sh "bash -c 'ulimit -c 0'"
```

Comment:
Default Jenkins interpreter is Bash. Why do you need to explicitly use it to invoke the command?